rustc_codegen_spirv/src/builder/builder_methods.rs

```diff
@@ -568,6 +566,18 @@ impl<'a, 'tcx> BuilderMethods<'a, 'tcx> for Builder<'a, 'tcx> {
     simple_uni_op! {neg, s_negate}
     simple_uni_op! {fneg, f_negate}
 
+    fn exactudiv(&mut self, lhs: Self::Value, rhs: Self::Value) -> Self::Value {
+        let result = self.udiv(lhs, rhs);
+        self.zombie(result.def, "exactudiv is not supported yet");
```

Comment:
You don't need a zombie here. `exactudiv` is the same as `udiv` except that it is UB when there is a remainder and in a few other cases. It is fine to just ignore the UB. You may want to add a FIXME to take advantage of the UB in the future.

Reply:
ah! neat :sparkles: